app/Enums/UserPlatform.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enums;
+
+use Spatie\TypeScriptTransformer\Attributes\TypeScript;
+
+/**
+ * This is distinct from the `Platform` entities we hold in the
+ * DB. Those entities are associated with emulators. This enum
+ * is used for pattern-matching against the user's browser
+ * user agent so we can concretely trigger certain behaviors
+ * conditionally based on whatever we detect.
+ */
+#[TypeScript]
+enum UserPlatform: string
```

Comment:
Would `ClientPlatform`, `ClientOS`, or `UserOS` work better?

Reply:
I like `UserOS` the most - renamed in latest.